src/main/java/io/github/thebusybiscuit/exoticgarden/PlantsListener.java

```diff
@@ -123,11 +122,9 @@ public void onGenerate(ChunkPopulateEvent e) {
 
         if (!cfg.getStringList("world-blacklist").contains(e.getWorld().getName())) {
             Random random = ThreadLocalRandom.current();
-
             if (random.nextInt(100) < cfg.getInt("chances.BUSH")) {
                 Berry berry = ExoticGarden.getBerries().get(random.nextInt(ExoticGarden.getBerries().size()));
                 if (berry.getType().equals(PlantType.ORE_PLANT)) return;
-
```

Comment:
Not really sure what the point of removing these lines were. Helps readability. Removing them does nothing but harm.